docs/source/tutorials_source/platform/tutorial_cropped_objects_metadata.py

```diff
@@ -0,0 +1,166 @@
+
+    In object detection applications, it can happen that the detector does not detect an object
+    because it did not see any examples of this or similar object yet.
+    This especially a problem in warehouse and retail applications,
+    as new products get added to the shelves every day. These new products with new appearence
+    often have a low so-called objectness score. Currently, finding these objects false detected
+    as non-objects needs a lot of manual labour, especially when having 1000s of new images coming
+    in every day. Lightly can make this work easier in a two-step approach:
+
+    1. A human finds one false negative and adds it to the missing examples.
+    2. Lightly finds all similar images which are also false negatives.
+    Thus it can propose to directly also add them as missing examples.
+
+    If there are e.g. 9 similar images for each missing example found by a human,
+    Lightly can speed up the process by a factor of 10.
+
+What you will learn
+--------------------
+
+* How to use an object detection model to crop objects out of a full image and save them as images.
+* How to save the object detection scores as metadata in a Lightly format.
+* How to upload the cropped images with their corresponding metadata to the webapp.
+* How to use the webapp to find false negatives and similar examples easily.
+
+Requirements
+-------------
+
+You can use your own dataset or the one we provide for this tutorial. The dataset
+we will use is the `SKU110k dataset <https://github.com/eg4000/SKU110K_CVPR19>` showing store shelves.
+The tutorial is computationally expensive if you run it on the full 110k images of products,
+thus we recommend running it on a subset of the dataset.
+
+For this tutorial the `lightly` pip package needs to be installed:
+
+.. code-block:: bash
+
+    # Install lightly as a pip package
+    pip install lightly.
+
+Steps
+-------------
+
+The steps of this tutorial are quite straightforward:
+    1. Define the dataset and torch dataloader for your dataset.
+    2. Define a pretrained object detection model. We use the retina net trained on COCO 2017. As it was not pretrained on a retail dataset, its performance is not state-of-the art. Nonetheless, it is sufficient for this tutorial and very easy to use.
+    3. Predict with the model on the dataset.
+    4. Use the bounding boxes of the object predictions to crop the objects out of the full images and save them in the output directory.
+    5. Extract the objectness scores and save them as custom metadata in a .json file.
+    6. Use the lightly-magic command to upload the cropped images, their embeddings and the objectness scores.
+    7. In the Lightly Webapp: Configure the objectness score as custom metadata.
+    8. In the Lightly Webapp: Sort the images in the explore view by increasing objectness score. This allows to easily find missing examples / false positives and similar images to them.
+
+
```

Comment:
Can we add somewhere the expected dataset format? Should there be subfolders? Are these just images (e.g. png, jpg?)

Reply:
The dataset is just images. They can be in subfolders, but don't need to.